### ISSUE TITLE
Log full with "Event status: Delivered"

### ISSUE THREAD

**Issue by sooslaca** [vectordotdev/vector#10188]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Vector Version

```
vector 0.18.0 (x86_64-unknown-linux-musl c77e085 2021-11-18)
```

### Vector Configuration File

```toml
[sinks.elasticsearch]
  type              = "elasticsearch"
  inputs            = [ "syslog" ]
  endpoint          = "https://elasticsearch:443"
  tls.ca_file       = "/vector-ca.crt"
  tls.crt_file      = "/vector.crt"
  tls.key_file      = "/vector.key"
  buffer.max_events = 100000
  buffer.when_full  = "drop_newest"
  buffer.type       = "memory"
  index             = "${VECTOR_INDEX}"
  bulk_action       = "create"
  auth.strategy     = "basic"
  auth.user         = "vector"
  auth.password     = "vector"

[sinks.s3_dest]
  type                   = "aws_s3"
  inputs                 = [ "syslog" ]
  region                 = "us-east-1"
  bucket                 = "logs"
  key_prefix             = "${S3_PREFIX}/${HOSTNAME}/%Y-%m-%d/"
  compression            = "gzip"
  encoding               = "ndjson"
  batch.max_bytes        = 10000000                  # 10mb uncompressed
  batch.timeout_secs     = 300
  auth.access_key_id     = "akey"
  auth.secret_access_key = "skey"

```

### Debug Output
Log is full with:
```
Event status: Delivered
```

### Expected Behavior
be quiet, like in version 0.16.1

### Additional Context
Using docker based vector. Upgraded from 0.16.1 -> 0.18.0.  
After the upgrade docker logs are full with message
```
Event status: Delivered
```
I cannot find in the documentation how I can make it back to quiet.

**Comment by hhromic** [comment]
Hi, this has been fixed in https://github.com/vectordotdev/vector/pull/10117
This fix is scheduled for Vector `0.18.1`.

**Comment by sooslaca** [comment]
Thank you and sorry I was not able to find this one. Let me close the issue then.
